Masinette add kitemaker pagertree (#9747)
getsentry/sentry-docs: docs/product/integrations/notification-incidents/pagertreeintegration/index.mdx

---
title: PagerTree
sidebar_order: 1

> description: "Learn about Sentry's PagerTree integration. Convert issues and errors into SMS, Voice, Push, Email, and Slack notifications with PagerTree!"

Reviewer: ```suggestion
description: "Learn how to add Sentry's PagerTree integration and convert issues and errors into SMS, Voice, Push, Email, and Slack notifications with PagerTree."
```

---

> 
> This integration is maintained and supported by the company that created it. For more details, or questions, please feel free to reach out to support@pagertree.com.
> ## Install and Configure
> 

Reviewer: ```suggestion

This integration is maintained and supported by the company that created it. Feel free to reach out to support@pagertree.com if you need additional information or have any questions.

## Install and Configure

```

<Note>

Sentry owner, manager, or admin permissions are required to install this integration.

</Note>

> 1. Navigate to **Settings > Integrations > PagerTree**

Reviewer: ```suggestion
1. Go to [Sentry](https://sentry.io/) and click on **Settings > Integrations > PagerTree**
```
